app.py

```diff
@@ -213,11 +232,11 @@ def train_input():
         with gr.Box():
             gr.Markdown('''
             碰到抓狂的错误或者计算资源紧张的情况下，推荐直接在[NoteBook](https://modelscope.cn/my/mynotebook/preset)上按照如下命令自行体验
-            1. git clone https://www.modelscope.cn/studios/CVstudio/cv_human_portrait.git
-            2. cd cv_human_portrait
-            3. pip install -r requirements.txt 
-            4. pip install gradio==3.35.2
-            5. python app.py
+            (If any error occurs or the waiting queue is too long, please use our [NoteBook](https://modelscope.cn/my/mynotebook/preset) with the following commands)
```

Comment:
You may consider use ModelScope Notebook directly, by following the steps below: